integrations/dialpad/syncs/users.ts

```diff
@@ -0,0 +1,34 @@
+import type { NangoSync, ProxyConfiguration } from '../models';
+import type { DialpadUser, User } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    
+    const proxyConfiguration: ProxyConfiguration = {
+ 
+        // https://developers.dialpad.com/reference/userslist
+        endpoint: '/users',
```

Comment:
The endpoint should make the full request which would be `https://dialpad.com/api/v2/users`

The dialpad provider provides the base url as `https://dialpad.com` see https://github.com/NangoHQ/nango/blob/master/packages/shared/providers.yaml#L1989 so the `/api/v2` needs to be provided